charts/virtaitech-orion-vgpu/v1.0.0/templates/clusterrole.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: orion-scheduler-cluster-admin
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I think it's better to define a new cluster role for orion-scheduler instead of using cluster-admin.

Reply:
`orion-scheduler` is a fully functional k8s scheduler (it actually uses hyperkube to bootstrap a standard k8s scheduler). I think it is reasonable to use cluster-admin for orion-scheduler.

Reply:
👌